### ISSUE TITLE
replacement start button causes problems

### ISSUE THREAD

**Issue by greydutch** [Open-Shell/Open-Shell-Menu#1860]
### Describe the bug

I picked a third party start button as the original button shows a tooltip that is in the way (see https://github.com/Open-Shell/Open-Shell-Menu/issues/1307).

The new button looks fine but  when I right-click on it it does now show the Open Shell menu. Instead the taskbar briefly changes its color (to purple) and then goes back to the original color. There is no way to enter the Open Shell settings when using an alternative button, at least not in my case. The good thing is that the tooltip for this button can be deleted.

Thank you.

### Area of issue

Start menu

### To reproduce

1 Go to Start button
2 Right click on Start button
3 Taskbar behaves strangely.

### Expected behavior

I would either like to have the option to use the regular start button without tooltip or a third party button with the option to right-click on it for acces to the Open Shell menu.

[start button.pdf](https://github.com/Open-Shell/Open-Shell-Menu/files/14472350/start.button.pdf)

### Open-Shell version

4.4.191

### Windows version

Windows 11 23H2, Build 22631.3155

**Comment by ge0rdi** [comment]
> when I right-click on it it does not show the Open Shell menu

Does this happen also with original Start butto (`Replace Start Button` unchecked)?

**Comment by greydutch** [comment]
No, it doesn't, only when I use a replacement button. So at the moment, it is either the original button with the annoying tooltip or an alternative button which prevents right-click access to the Open Shell settings.
